src/components/Callout/styles.tsx

```diff
@@ -18,7 +18,7 @@ export const CalloutOuter = styled.div<CalloutProps>((props) => {
     margin-bottom: 0;
     margin-top: 1rem;
 
-    &:first-child {
+    &:first-of-type {
```

Comment:
Tests were complaining about using `:first-child` when using server side rendering and suggested to use `:first-of-type` instead.

Reply:
Theoretically, an HTML tag inside a `<Callout>` could be affected by this, although the only real consequence I see here is imagining having headings inside.  I don't really understand why `:first-of-type` is preferred, since I'm imagining that the issue with server-side rendering is that the order of elements in the DOM isn't guaranteed, right?  So in theory the two would have the same issue...  But if it makes the error go away, I'm tentatively in favor.

Reply:
Yes, I think the error is coming from `emotion`. From what I've read it looks like they are injecting the actual `<style>` tags as the first child which breaks `:first-child`. More info from here: https://github.com/emotion-js/emotion/issues/1178

Still looking into what to do about it though. I don't think there were any warnings for this before, but I'm only getting it during the tests so it could be related to Jest

Reply:
Adding @jakeburden too

It looks like there are some options to get around this issue here: https://github.com/emotion-js/emotion/issues/1105#issuecomment-557726922

I've tried option 1 and option 2 and they make the tests pass while still keeping the css for the component the same.   

When navigating to a page with a `Callout` on my localhost, I saw that the same error regarding the `:first-child` issue is also printed in the console, so it looks like this might've been an issue for a while but hasn't broken anything I think. Have either of you seen this before?

Regarding option 2, I was able to make the changes to tests and it looks like this:
```typescript
import Callout from '../styles';
import { render, screen } from '@testing-library/react';
import createCache from '@emotion/cache';
import { CacheProvider } from '@emotion/react';

function renderWithProvider(ui) {
  const myCache = createCache({ key: 'my-cache-key' });
  myCache.compat = true;

  const Wrapper = ({ children }) => (
    <CacheProvider value={myCache}>{children}</CacheProvider>
  );

  return render(ui, { wrapper: Wrapper });
}

describe('Callout', () => {
  test('renders info style callout when no warning prop is passed', () => {
    const { container } = renderWithProvider(
      <Callout>Text for callout</Callout>
    );

    const callout = container.children[0];

    expect(callout).toHaveStyle(`
      display: block;
      padding-left: 0.75rem;
      margin-bottom: 1rem;
      background-color: var(--color-ink-hv);
    `);
  });

  test('renders warning style callout when no warning prop is passed', () => {
    const { container } = renderWithProvider(
      <Callout warning>
        <div>Warning text for callout</div>
      </Callout>
    );

    const callout = container.children[0];

    expect(callout).toHaveStyle(`
      display: block;
      padding-left: 0.75rem;
      margin-bottom: 1rem;
      background-color: var(--color-orange-hv);
    `);
  });
});
```
From my understanding of option 2, the `CacheProvider` and the `compat` flag store the styles in a cache and so they aren't inserted into the style elements. That's why the issue with `:first-child` isn't thrown anymore. (source: https://github.com/emotion-js/emotion/issues/2049#issuecomment-713868664)